src/d3a/setup/area_events/pv_event.py

```diff
@@ -0,0 +1,48 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+from d3a.models.appliance.switchable import SwitchableAppliance
+from d3a.models.area import Area
+from d3a.models.area.events import StrategyEvents
+from d3a.models.strategy.pv import PVStrategy
+from d3a.models.appliance.pv import PVAppliance
+from d3a.models.strategy.load_hours import LoadHoursStrategy
+
+
+def get_setup(config):
```

Comment:
We used to have a short description of expected behaviour inside a setup file. Shouldn't we restart adding short description of expected behaviour?

Reply:
I thought that the expected behavior is clear here, that's why I did not add it. The setup file consists of just one load and one PV, and on the PV there is an event that increases the number of panels from 1 to 10, and the max panel power from 160 to 320. This has the effect that the max power of the PV is multiplied by 20. Its purpose is to use it in the integration test to validate that the event actually triggers this increase in PV power.